tests/test_modeling_common.py

```diff
@@ -1727,28 +1728,81 @@ def assert_almost_equals(self, a: np.ndarray, b: np.ndarray, tol: float):
         diff = np.abs((a - b)).max()
         self.assertLessEqual(diff, tol, f"Difference between torch and flax is {diff} (>= {tol}).")
 
+    def check_outputs(self, fx_outputs, pt_outputs, model_class, names):
+        """
+        Args:
+            model_class: The class of the model that is currently testing. For example, ..., etc.
+            Currently unused, but it could make debugging easier and faster.
+
+            names: A string, or a list of strings. These specify what fx_outputs/pt_outputs represent in the model outputs.
+                Currently unused, but in the future, we could use this information to make the error message clearer
+                by giving the name(s) of the output tensor(s) with large difference(s) between PT and Flax.
+        """
+        # `bigbird_block_sparse_attention` in `FlaxBigBird` returns `attention_probs = None`, while in PyTorch version,
+        # an effort was done to return `attention_probs` (yet to be verified).
+        if type(names) == str and names.startswith("attentions"):
```

Comment:
I decided not to add attentions to FlaxBigBird at this moment, and just skip this test case for now.

Reply:
That's a bit too hacky for me here. Can't we just skip the test in `test_modeling_big_bird.py` instead? I'm not a fan of cluttering the core testing file with such if statements

Reply:
Currently, `test_modeling_big_bird.py` doesn't overwrite the PT common PT/Flax equivalence tests.
I will try to come up with a better solution.

Reply:
I overwrite the `check_outputs` in `BigBirdModelTest` with a call to `super()` - tried to make the change minimal.